atqa-framework/utils/copyDirHandler.ts

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs-extra';
+import { Folder } from '../interfaces/Folder.interface';
+
+export const copyDir = async (dir: Folder): Promise<void> => {
+  try {
+    fs.copySync(dir.sourcePath, dir.destinationPath, {
+      overwrite: true,
+      recursive: true,
+    });
+    console.log('Dir has been copied');
+  } catch (err) {
+    console.log(`${err} has found`);
```

Comment:
So if copying fails, calling code continues as normal?

Reply:
It will be rewritten on the next stage.  I suggest keep it as is until the coming changes.